apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/instanceQuestion.ts

```diff
@@ -79,7 +79,7 @@ router.get(
       instance_question_id: res.locals.instance_question.id,
       variant_id: req.params.variant_id,
       urlPrefix: res.locals.urlPrefix,
-      questionContext: null,
```

Comment:
Bug? 👀

Reply:
Kinda, but not quite. I changed the type of this function in question-render.ts to be QuestionContext (without null/undefined), mostly to keep it strict. In its previous implementation, the question context was only passed to the score panels, so the calls for on demand submission rendering (which don't render score panels) did not need the question context, so null was ok. That said, a proper translation of submission.ejs turns out that the question context is used in that context, though not in a way that would affect this particular user case: it is used to determine the URL for dynamic retrieval, and since this case is already being retrieved dynamically, the URL is not needed. In other words, a strict type is useful, but missing it would not cause a problem. Same for the other calls.